test/grisp_connect_test.hrl

```diff
@@ -0,0 +1,28 @@
+%--- Async asserts -------------------------------------------------------------
+
+% Spawn a process to wait for the result and check the result using assert.
+-define(asyncAssertEqual(Pattern, M, F, A),
+        spawn_monitor(?asyncAssertEqualFun(Pattern, M, F, A))).
```

Comment:
Shouldn't we do a spawn_link, so if the spawned process crashes it will crash the test ?

Reply:
You mean if the test crashes the spawned process crashes? That would make sense. We have to trap_exist because the spawned process will exit for sure, it just should be normal status.

Reply:
If the process is waiting for the call to `?matchAsyncCall` to go ve the result that should be matched, it will never exit normally before the call, so it is safe to not trap exit. We just have to unlink and monitor in `?assertAsyncCall`